src/helpers/Mailer/templates/index.js

```diff
@@ -0,0 +1,5 @@
+import notifications from './notifications';
```

Comment:
I thought about the other modules that will come after. I thought it will be nice to import all of them. But if you think that we don't really need it, I will remove it.

Reply:
I just removed the one in ```helpers/tokens```